examples/memodel/memodel.py

```diff
@@ -0,0 +1,402 @@
+        plot_bAP_EPSP=access_point.pipeline_settings.plot_bAP_EPSP,
+        plot_dendritic_ISI_CV=True,  # for detailed cADpyr cells. will be skipped otherwise
+        plot_dendritic_rheobase=True,  # for detailed cADpyr cells. will be skipped otherwise
+        only_validated=False,
+        save_recordings=False,
+        load_from_local=False,
+        cell_evaluator=cell_evaluator,  # <-- feed the modified evaluator here
+    )
+
+    return emodel_score
+
+
+def get_nexus_images(access_point, seed, new_emodel_metadata, morph_id, emodel_id):
+    """Get the nexus images from memodel method using new emodel metadata."""
+    # create MEModel (easier to get images with it)
+    memodel = MEModel(
+        seed=seed,
+        emodel_metadata=access_point.emodel_metadata,
+        emodel_id=emodel_id,
+        morphology_id=morph_id,
+        validated=False,
+    )
+
+    # update MEModel metadata
+    memodel.emodel_metadata = new_emodel_metadata
+
+    return memodel.build_pdf_dependencies(memodel.seed)
+
+
+def update_memodel(
+    forge,
+    memodel_r,
+    seed,
+    new_emodel_metadata,
+    subject_ontology,
+    brain_location_ontology,
+    nexus_images,
+    emodel_score=None,
+    new_status="done",
+):
+    """Update ME-Model."""
+    # update metadata in resource
+    metadata_for_resource = new_emodel_metadata.for_resource()
+    metadata_for_resource["subject"] = subject_ontology
+    metadata_for_resource["brainLocation"] = brain_location_ontology
+    for key, item in metadata_for_resource.items():
+        setattr(memodel_r, key, item)
+
+    memodel_r.seed = seed
+    memodel_r.objectOfStudy = {
+        "@id": "http://bbp.epfl.ch/neurosciencegraph/taxonomies/objectsofstudy/singlecells",
+        "label": "Single Cell",
+    }
+    memodel_r.status = new_status
+    if emodel_score is not None:
+        memodel_r.score = emodel_score
+
+    # do not add any description: we expect it to be already present
+
+    # make memodel resource into a Dataset to be able to add images
+    # have store_metadata=True to be able to update resource
+    memodel_r = Dataset.from_resource(forge, memodel_r, store_metadata=True)
+
+    # add images in memodel resource
+    # Do NOT do this BEFORE turning resource into a Dataset.
+    # That would break the storing LazyAction into a string
+    for path in nexus_images:
+        resource_type = path.split("__")[-1].split(".")[0]
+        memodel_r.add_image(
+            path=path,
+            content_type=f"application/{path.split('.')[-1]}",
+            about=resource_type,
+        )
+
+    # update memodel resource
+    forge.update(memodel_r)
+
+
+if __name__ == "__main__":
+    project = "mmb-point-neuron-framework-model"
+    organisation = "bbp"
+    endpoint = "https://bbp.epfl.ch/nexus/v1"
+    forge_path = "./forge.yml"  # this file has to be present
+    forge_ontology_path = "./forge_ontology_path.yml"  # this file also
+    memodel_id = "<MEMODEL ID>"
```

Comment:
Could you just add a comment in the script here to say to change the `memodel_id`?

Reply:
https://github.com/BlueBrain/BluePyEModel/blob/341b3bfe05801c58a8324e0cd2a87f7009c1db42/examples/memodel/memodel.py#L287